src/components/modals/AlertModal/AlertModal.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+AlertModal.propTypes = {
+  children: PropTypes.string,
+  handleOnOk: PropTypes.func,
+  handleOnCancel: PropTypes.func,
+}
+
+function AlertModal({ children, handleOnOk, handleOnCancel }) {
+  return (
+    <ErrorWrapper>
+      <ErrorModal>
+        <ErrorMessage>ACHTUNG</ErrorMessage>
+        <InfoMessage>{children}</InfoMessage>
+        <ActionWrapper>
+          <Action type="cancel" color="" onClick={handleOnCancel}>
+            NEIN
+          </Action>
+          <Action type="confirm" onClick={handleOnOk}>
+            JA
+          </Action>
+        </ActionWrapper>
+      </ErrorModal>
+    </ErrorWrapper>
+  )
+}
+
+export default AlertModal
+
+const ErrorWrapper = styled.div`
+  background-color: rgba(0, 0, 0, 0.7);
+  bottom: 0;
+  height: auto;
+  left: 0;
+  margin: 0 auto;
+  overflow: auto;
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: 100%;
+  z-index: 100;
+`
+
+const ErrorModal = styled.section`
+  background-color: #f3f3f3;
+  border-radius: 10px;
+  border: 1px solid #ccc;
+  display: flex;
+  flex-direction: column;
+  height: 18em;
+  justify-content: space-between;
+  left: 50%;
+  margin-left: -10em;
```

Comment:
Our coach said that using em for anything expect font-sizes such as margins, padding etc. is discouraged. You could try using px or %. Or leave it as it is if it works as expected on every screen size.

Reply:
I know, will change that when working on the final design. this is only proof of concept style code. i need to change a lot of style code anywhere.